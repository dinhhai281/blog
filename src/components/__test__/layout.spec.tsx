import Layout from '@components/layout';
import { render } from '@utils/testing';
import React from 'react';

describe('Layout', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Layout></Layout>);

    expect(asFragment()).toMatchSnapshot();
  });
});
