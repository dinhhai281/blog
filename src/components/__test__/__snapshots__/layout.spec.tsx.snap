// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container px-5 pt-8 mx-auto"
  >
    <header
      class="flex justify-between items-center mb-8"
    >
      <h1
        class="text-pink-700 text-4xl font-bold leading-10"
      >
        <a
          href="/en/about/"
        >
          Onyxia
        </a>
      </h1>
      <div>
        Â 
      </div>
    </header>
  </div>
</DocumentFragment>
`;
